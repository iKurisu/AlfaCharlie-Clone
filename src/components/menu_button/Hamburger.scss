.menu-hamburger line {
  fill: none;
  stroke: #3a3c48;
  stroke-width: 5;
}
