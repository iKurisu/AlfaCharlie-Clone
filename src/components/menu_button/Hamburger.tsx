import React from "react";
import "./Hamburger.scss";

const Hamburger = (): JSX.Element => (
  <svg className="menu-hamburger" x="0" y="0" viewBox="0 0 72 72">
    <line x1="2" y1="9" x2="69" y2="9"></line>
    <line x1="2" y1="35" x2="69" y2="35"></line>
    <line x1="2" y1="61" x2="69" y2="61"></line>
  </svg>
);

export default Hamburger;
