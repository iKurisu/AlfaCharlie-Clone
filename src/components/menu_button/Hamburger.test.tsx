import React from "react";
import renderer from "react-test-renderer";
import Hamburger from "./Hamburger";

it("renders correctly", (): void => {
  const hamburger: renderer.ReactTestRenderer = renderer.create(<Hamburger />);

  expect(hamburger).toMatchSnapshot();
});
