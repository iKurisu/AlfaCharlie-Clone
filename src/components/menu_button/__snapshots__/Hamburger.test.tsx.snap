// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<svg
  className="menu-hamburger"
  viewBox="0 0 72 72"
  x="0"
  y="0"
>
  <line
    x1="2"
    x2="69"
    y1="9"
    y2="9"
  />
  <line
    x1="2"
    x2="69"
    y1="35"
    y2="35"
  />
  <line
    x1="2"
    x2="69"
    y1="61"
    y2="61"
  />
</svg>
`;
